wsd/DocumentBroker.cpp

```diff
@@ -834,6 +835,12 @@ bool DocumentBroker::isAlive() const
     return _childProcess && _childProcess->isAlive();
 }
 
+void DocumentBroker::timeoutNotLoaded(const std::chrono::steady_clock::time_point& now)
```

Comment:
FYI, `time_point` is a single word (8 bytes) and is best passed by value.

I have a PR that converts all such cases from by-ref to by-val: #11427